source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/navigation.tsx

```diff
@@ -91,7 +91,7 @@ class Navigation extends React.Component<IProps & i18nProps> {
           <RequireRole roleName={ROLE.OrganizationAdmin}>
             <MenuItem
               name={t('sidebar.organizationSettings')}
-              to={`/organizations/${currentOrganizationId}/settings`}
+              to={`/organizations/${currentOrganizationId}/settings/basic-info`}
```

Comment:
this is not related to this issue, but a new bug that I found while navigation to organization settings. This happened because I changed the Sidebar menu for super admin.

Reply:
what was the bug?

Reply:
so what happens when you visit `organizations/id/settings/`?

Reply:
several routes get mark as active because in responsiveNav I changed it to
`isActive={() => location.pathname.startsWith(i.to)}` because I wanted to support super admin flow where we have this type or url "/admin/settings/org" and "/admin/settings/org/new" but in organization settings basic-info get mark as active because every route contains /org/settings

Reply:
organizations/id/settings/ will show blank page... so I guess this was not ok to change it I need to think a solution for admin/settings too

Reply:
why not change your isActive to be an exact match? and use pattern matching? 
obvs not now, but after MVP?